FBTweak/_FBTweakCollectionViewController.m

```diff
@@ -28,11 +30,22 @@ - (instancetype)initWithTweakCategory:(id<FBTweakCategory>)category
     _tweakCategory = category;
     self.clearsSelectionOnViewWillAppear = YES;
     self.title = _tweakCategory.name;
-    self.refreshControl = [[UIRefreshControl alloc] init];
-    [self.refreshControl addTarget:self
-                            action:@selector(_reloadData)
-                  forControlEvents:UIControlEventValueChanged];
-    [self _reloadData];
+
+    _kvoController = [FBKVOController controllerWithObserver:self];
+    [_kvoController observe:_tweakCategory keyPath:@"tweakCollections"
+                        options:NSKeyValueObservingOptionInitial | NSKeyValueObservingOptionNew
+                          block:^(id observer, id object, NSDictionary *change) {
+                            dispatch_async(dispatch_get_main_queue(), ^{
+                              [self _reloadData];
+                            });
+    }];
+
+    if ([self _supportsUpdates]) {
+      self.refreshControl = [[UIRefreshControl alloc] init];
+      [self.refreshControl addTarget:self
+                              action:@selector(_updateCategory)
```

Comment:
`action:...` can be moved to the preceding line.

Reply:
done